recipes/ms2query/meta.yaml

```diff
@@ -21,19 +21,19 @@ build:
 
```

Comment:
I think you will need to bump `build > number` (line14) too

Reply:
Thanks. I tried, but the linting did not pass if I set build number to 1.